packages/bento-design-system/src/AreaLoader/AreaLoader.tsx

```diff
@@ -54,28 +53,24 @@ export type { Props as AreaLoaderProps };
  */
 export function AreaLoader({ message }: Props) {
   const config = useBentoConfig().areaLoader;
+
   return (
     <Box
-      className={clsx(
-        container,
-        bentoSprinkles({
-          background: config.scrimColor === "dark" ? "backgroundDarkScrim" : "backgroundLightScrim",
-        })
-      )}
+      background={config.scrimColor === "dark" ? "backgroundDarkScrim" : "backgroundLightScrim"}
+      className={container}
```

Comment:
it didn't event make sense to use `sprinkles` directly here, since we can just use `Box` (which internally already uses `sprinkles` from the context)

Reply:
basically it makes sense to use `sprinkles` directly only if we can't use `Box` directly, and the only case we have right now is the custom styling of react-select components.